projects/htop.dev/package.yml

```diff
@@ -0,0 +1,29 @@
+distributable:
+  # url: https://github.com/htop-dev/htop/archive/refs/tags/{{version}}.tar.gz
+  url: https://github.com/htop-dev/htop/releases/download/{{version}}/htop-{{version}}.tar.xz
+  strip-components: 1
+
+versions:
+  github: htop-dev/htop/tags
+  strip: /v/
+
+dependencies:
+  invisible-island.net/ncurses: 6
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    gnu.org/autoconf: '*'
+    gnu.org/automake: '*'
+    invisible-island.net/ncurses: 6
+  script: |
+    ./autogen.sh && \
```

Comment:
they don’t provide a tarball with `./configure` already generated?

Reply:
Yeah so probs you don’t need to do the `autogen.sh`

Reply:
my bad I misread your question. `configure.ac` is provided so we do need `autogen.sh`

Reply:
There's a `configure`, please remove the autogen and deps:
<img width="923" alt="image" src="https://user-images.githubusercontent.com/58962/202211399-b336a2f2-d6e9-46cd-b4c6-0f3309e62e10.png">